forte/data/ontology/configs/stanfordnlp_ontology.json

```diff
@@ -1,24 +1,20 @@
 {
-  "ontology_name": "forte.data.ontology.stanfordnlp_ontology",
-  "imports": [
-    "forte.data.ontology.top",
-    "forte.data.ontology.base_ontology"
-  ],
+  "ontology_name": "stanfordnlp_ontology",
   "entry_definitions": [
     {
-      "entry_name": "forte.data.ontology.stanfordnlp_ontology.Token",
-      "parent_entry": "forte.data.ontology.top.Annotation",
+      "entry_name": "stanfordnlp_ontology.Token",
+      "parent_entry": "Annotation",
       "attributes": [
         {
           "attribute_name": "lemma",
           "attribute_type": "str"
         },
         {
-          "attribute_name": "pos_tag",
-          "attribute_type": "str"
+          "attribute_name": "is_verb",
```

Comment:
Why are the example attributes copied here?

Reply:
restored it - sorry about this, it was an error.